static/js/components/EducationDialog.js

```diff
@@ -8,21 +9,22 @@ import ProfileFormFields from '../util/ProfileFormFields';
 import { educationValidation } from '../util/validation';
 
 export default class EducationDialog extends ProfileFormFields {
-  constructor(props) {
+  constructor(props: {}) {
```

Comment:
I don't think so - some things have subtle differences, like `Number` vs `number`, but I think we can prefer `Object` without it making a difference.